src/main/java/dev/ai4j/openai4j/RequestDisablerInterceptor.java

```diff
@@ -0,0 +1,21 @@
+package dev.ai4j.openai4j;
+
+import okhttp3.Interceptor;
+import okhttp3.Protocol;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+public class RequestDisablerInterceptor implements Interceptor {
+    private static final String MESSAGE = "Requests to the live system are disabled";
+
+    @Override
+    public Response intercept(Chain chain) {
+        return new Response.Builder()
+            .request(chain.request())
+            .protocol(Protocol.HTTP_1_1)
+            .code(400)
```

Comment:
Would it make sense to return more specific code?
400 can be returned if something is wrong with the request itself.

Reply:
I thought about that. The code needs to be somewhere in the 4xx in order for it to be treated as an error condition (and have the `OpenAiHttpException` get thrown. None of the other 4xx error codes seemed to fit this use case, so I picked 400.

Reply:
> Would it make sense to return more specific code

Maybe, finally, a use case for 418 (I am a teapot)? :D